src/events/openwhisk.js

```diff
@@ -31,24 +31,52 @@ function openWhiskWrapper(wrappedFunction) {
         ]);
 
         invokeEvent.setResource(resource);
+        eventInterface.addToMetadata(invokeEvent, {
+            params: options.params,
```

Comment:
maybe let's pass this as the third argument? (so it won't be recorded when metadataOnly=True)

Reply:
but then it is not consistent with the data that is added in the runner. there all the params are added to the default map:
https://github.com/epsagon/epsagon-node/blob/abcf003c2ab88170e018f9aba650d537d1cb6332/src/wrappers/openwhisk.js#L73-L79